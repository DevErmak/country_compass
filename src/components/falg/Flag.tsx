import './flag.css';

type Props = {
  flags: string;
  flagsAlt: string;
  coatOfArms: string;
};

export default function Flag({ flags, flagsAlt, coatOfArms }: Props) {
  return (
    <div className="container-flag">
      <div className="flag">Flag</div>
      <img className="flag-img" src={flags} alt={flagsAlt} />
      <div className="coat-of-arms">Coat of arms</div>
      <img className="coat-of-arms-img" src={coatOfArms} alt={''} />
    </div>
  );
}
