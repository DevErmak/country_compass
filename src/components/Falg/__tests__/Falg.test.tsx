import renderer from 'react-test-renderer';
import Flag from '../Flag';

it('renders correctly', () => {
  const flags = '';
  const flagsAlt = '';
  const coatOfArms = '';
  const component = renderer
    .create(<Flag flags={flags} flagsAlt={flagsAlt} coatOfArms={coatOfArms} />)
    .toJSON();
  expect(component).toMatchSnapshot();
});
