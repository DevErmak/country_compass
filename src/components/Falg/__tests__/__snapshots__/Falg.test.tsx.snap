// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container-flag"
>
  <div
    className="flag"
  >
    Flag
  </div>
  <img
    alt=""
    className="flag-img"
    src=""
  />
  <div
    className="coat-of-arms"
  >
    Coat of arms
  </div>
  <img
    alt=""
    className="coat-of-arms-img"
    src=""
  />
</div>
`;
